grr/server/grr_response_server/gui/ui/components/client_details/entry_history_dialog/entry_history_dialog.ts

```diff
@@ -0,0 +1,60 @@
+import {Component, Inject} from '@angular/core';
+import {MAT_DIALOG_DATA} from '@angular/material/dialog';
+import {Client} from '@app/lib/models/client';
+
+/** Entry type */
+export type EntryType =
+    'primitive'|'timestamp'|'size'|'user-list'|'interface-list'|'volume-list';
+
+/** Parameters required to open an EntryHistoryDialog */
+export interface EntryHistoryDialogParams {
+  readonly path: string;
```

Comment:
Judging from the code that uses it (`data.path.split('.')`) this is some list of components. Is there a reason to store it as a string rather than using a more structured way from the very beginning as a list of components?